### PR TITLE
Improve Cost Card style

### PR DESCRIPTION
**Pull request by kula1922**
- Move cost to right
- Space as separator
- Add lines between rows and columns

### CODE REVIEW

**Comment by mkurek** [comment]
it's perfect method to write some unit tests :)

File: src/ralph_scrooge/media/scrooge/partials/costcard.html [hunk 1]
```diff
@@ -3,13 +3,24 @@
...
+        <th data-field="cost" data-halign="center" data-align="right" data-sortable="true" data-formatter="costFormatter">Cost</th>
       </tr>
     </thead>
   </table>
   <script>
       function costFormatter(value, row) {
-        return value + ' PLN';
+        value_list = String(value).split('.');
+        new_value = '';
+        for(var i=value_list[0].length;i>0;i--) {
```
Comment (ar4s) [inline src/ralph_scrooge/media/scrooge/partials/costcard.html]: for (var i = value_list[0].length; i > 0; i--) {

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**